posts/2019-02-20-framework-release-v138-websockets.md

```diff
@@ -0,0 +1,161 @@
+---
+title: "Serverless Framework v1.38 - Introducing Websockets Support"
+description: "Check out the latest Serverless Framework v1.38 release."
+date: 2019-02-07
+thumbnail: 'https://s3-us-west-2.amazonaws.com/assets.blog.serverless.com/framework-updates/framework-v137-thumb.png'
+heroImage: 'https://s3-us-west-2.amazonaws.com/assets.blog.serverless.com/framework-updates/framework-v137-header.png'
+category:
+  - news
+authors:
+  - EslamHefnawy
+---
+
+### Introducing Websockets Support
+After years of waiting, [AWS has finally added support for websockets in API Gateway & Lambda](https://aws.amazon.com/blogs/compute/announcing-websocket-apis-in-amazon-api-gateway/), and because CloudFormation didn't have support for websockets at the time of the initial release, we've created an [external plugin](https://github.com/serverless/serverless-websockets-plugin) that lets you integrate websockets into you Serverless Framework projects.
```

Comment:
Should we call it an "external plugin" or maybe an "official plugin" to already establish the notion of "official" vs. "non-official"? /cc @shortjared

Reply:
good point